### ISSUE TITLE
Ground-bridge does not work if COFH core is installed

### ISSUE THREAD

**Issue by SoundwaveSuperior** [FiskFille/TransformersMod#212]
Hello, just wanted to report a frustrating issue I've encountered with COFH core (In other words: that mod that Thermal Expansion requires) and the Ground-bridge. So I was installing COFH core to fix an error I encountered involving some RF issues with another mod, and before I installed COFH I had a working Ground-bridge. I had used it to teleport myself, it was working fine. However, after installing COFH core the control panel no longer can detect my Ground-bridge frame, stating that there was no frame in range (despite it being on the same level next to an accurately constructed frame that was displaying the directional arrow, and having worked in this exact same combination before).

Any ideas on why this is or how I can fix it? I've uninstalled COFH core and deleted every file related to it in my minecraft installation, the same error is still occurring. I've tried everything, from upping the detection range in the config file to creating another world and building one there. Nothing works.

EDIT: Added screenshots of the issue from a world created AFTER I installed COFH core but also AFTER I deleted it. The blocks behind the control panel are purely so it is easier to see the error message, and are from Extra Utilities, which had no effect on the Ground-bridge as I had it and 24 other mods running alongside the TF mod and the Ground-bridge worked fine, as far as I can tell COFH core is the mod that caused the problem. The Ground-bridge is, as far as I can tell, accurately built, and I have had the arrow flipped both ways but nothing changed.

On another note, the control panel also does not appear to be accepting the power, could be due to COFH core as well but I have no idea there. 
![2017-04-22_20 52 00](https://cloud.githubusercontent.com/assets/27890943/25304635/f775c36c-279d-11e7-94ba-fa2aa6876dc8.png)
![2017-04-22_20 52 09](https://cloud.githubusercontent.com/assets/27890943/25304637/f88a0380-279d-11e7-90f2-42f585e61536.png)

**Comment by Tigerman277** [comment]
hey, I'm having the same issue but only the last part, it doesn't take power for me either, it recognizes that a frame is there and it can do it, but it won't take the power but I don't have the COFH core or thermal expansion, so my guess is that it's something else, if you could tell me the list of mods you have installed if any other than thermal expansion and the transformers mod, I'll check my mods and see if we have any in common and try deleting that one if any, honestly I'm no modder but I've dealt with errors before and I kinda have a feeling we'd be able to fix it on our own, alright have a good day man.

**Comment by Maks2103** [comment]
I have Thermal Expansion and TF mod installed, and everything works, so maybe you need to brake control panel and set it back down?
If reseting console will not work - try to break portal frame, and build it up again.
@SoundwaveSuperior @Tigerman277

**Comment by Maks2103** [comment]
If reseting console will not work - try to break portal frame, and build it up again.

**Comment by Tigerman277** [comment]
alright I'll try both

**Comment by Tigerman277** [comment]
ummm okay so either two things will fix it because it just started working, are you using regular energon or red energon to power the ground bridge and two, are you putting the frame one block into the ground

**Comment by Tigerman277** [comment]
I think the issue for me is that my ground bridge is underground but I'm not sure

**Comment by Tigerman277** [comment]
if there's something close to that build it on the surface, fill up the transmitter completely and charge it straight from the transmitter

**Comment by Tigerman277** [comment]
okay well have your ground bridge control panel on the ground and the frame's bottom matching the Y level of the control panel and make sure the portal isn't touching the ground, that's working for me don't ask why I don't know but it is, and if it doesn't work for you, well crap

**Comment by Tigerman277** [comment]
I just now noticed that the last message from someone else that wasn't me. wasn't from soundwave ._.

**Comment by SoundwaveSuperior** [comment]
@Maks2103 I tried that, completely breaking it and rebuilding it, having the controller level to the frame, breaking and replacing the controller, using both red and normal energon to power it, and repeating this process in multiple clean worlds, nothing works. And I did another test: I made a fresh minecraft installation with no mods other then Transformers. Built a space bridge, worked fine. Then I installed JUST COFH core, and made another space bridge in another world, and it stopped working, same error, same set up, just not detecting the frame.

And @Tigerman277, I doubt my mod list would help, considering that, as I stated, I had it working prior to installing COFH, and it was only when installing COFH that the issue occured, but here's my modlist anyway. COFH core is excluded from this list, as this is post uninstall of that, so just add that in to the list.

![screen shot 2017-04-23 at 9 04 56 am](https://cloud.githubusercontent.com/assets/27890943/25309595/fde04cc0-2803-11e7-8b29-dcac7bebf9d8.png)
![screen shot 2017-04-23 at 9 05 05 am](https://cloud.githubusercontent.com/assets/27890943/25309598/01e17bd2-2804-11e7-91ec-f04d992f7477.png)
![screen shot 2017-04-23 at 9 05 12 am](https://cloud.githubusercontent.com/assets/27890943/25309597/015bb7fe-2804-11e7-9096-f267e6188e8a.png)
![screen shot 2017-04-23 at 9 05 16 am](https://cloud.githubusercontent.com/assets/27890943/25309599/02ad56e4-2804-11e7-8331-78ddfaa948ec.png)

**Comment by Maks2103** [comment]
@SoundwaveSuperior , try this modpack: https://yadi.sk/d/gUiY0z_x3HF8ja

I created it myself, and everything works.
BTW, did you tried install both, Transformers mod and COFH Core, and then generate new world?

**Comment by SoundwaveSuperior** [comment]
@Maks2103 I'll try that modpack at some point, thanks for linking it, and yes, I did try that, it still didn't work.

**Comment by Maks2103** [comment]
@SoundwaveSuperior , I just noyiced, that you using fastcraft. Try to temporaly remove it.

**Comment by SoundwaveSuperior** [comment]
@Maks2103 It's not fastcraft, It it is definitely COFH core, I did some more testing and created a fresh version of minecraft with nothing but the Transformers mod, nothing else, not even not enough items, and a Ground-bridge ran fine there. Then I installed COFH core and it broke, thus confirming that is indeed COFH core causing the error.

**Comment by Maks2103** [comment]
@SoundwaveSuperior , so, can you try to install both, Transforemrs Mod and COFH core, and then generate world?

**Comment by Tigerman277** [comment]
wellI guess I can't help, I don't have the cofh core or thermal expansion so for me it's not that, I hope you manage to fix it, good luck man

**Comment by Tigerman277** [comment]
right as I say that it works for me, I have one last suggestion and it actually worked just now, you might already know this but I didn't know so, at the back of the controls there are dark-cyan squares, apparently that is where the energon goes so you need to have that facing the transmitter/relay and it should start to power, I'm stupid for not realizing it was important, maybe you didn't know it exists either, but try that and it might start charging if it does, great! if it doesn't, well good luck

**Comment by FiskFille** [comment]
Yes, when the energy transfer beam is red it means the beam can't reach the block's contact point. As for the COFH core, I will look into it once I get internet fixed at home.

**Comment by Optoclaw** [comment]
I have a similar issue where the ground bridge frame is detected, it is recieving power, and I have built it both one block in the ground as well as having the bottom of the frame level with the control panel, BUT the panel keeps telling me that adjacent blocks are blocking the frame from working when there is nothing for five blocks in front or behind it
